DiamondLightSource/python-artemis
# 62: Allow GDA to send filename and prefix separately

Fixes https://github.com/DiamondLightSource/python-artemis/issues/62

To test:
* Confirm unit tests pass
* Compare to prefix changes in https://github.com/DiamondLightSource/python-artemis/tree/local_i03_testing_changes and confirm that they would be correct
